src/game/frontend/submenus/Network.cpp

```diff
@@ -26,7 +26,8 @@ namespace YimMenu::Submenus
 		auto spoofing         = std::make_shared<Category>("Spoofing");
 		auto nameChangerGroup = std::make_shared<Group>("Name Changer", GetListBoxDimensions());
 		auto spoofingColumns  = std::make_shared<Column>(1);
-		session->AddItem(std::make_shared<CommandItem>("explodeall"_J));
+		session->AddItem(std::make_shared<CommandItem>("quitsession"_J));
+    session->AddItem(std::make_shared<CommandItem>("explodeall"_J));
```

Comment:
This might have been when I was resolving conflicts, but if this could be indented that would be great.